pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbcluster/basicsecondaryalloydbcluster/_http.log

```diff
@@ -913,73 +916,6 @@ X-Xss-Protection: 0
 
 ---
 
-GET https://alloydb.googleapis.com/v1beta/projects/${projectId}/locations/us-west1/clusters/alloydbcluster-1-${uniqueId}/instances/alloydbinstance-${uniqueId}?alt=json
```

Comment:
This GET is no longer there because in the TF-based controller, a GET of the target resource is always done after an operation is completed while in the direct controller we don't do it.